backend/src/main/kotlin/com/tngtech/apicenter/backend/connector/rest/controller/VersionController.kt

```diff
@@ -1,20 +1,50 @@
 package com.tngtech.apicenter.backend.connector.rest.controller
 
+import com.fasterxml.jackson.core.JsonProcessingException
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.dataformat.yaml.YAMLMapper
+
+import com.tngtech.apicenter.backend.domain.entity.Version
 import com.tngtech.apicenter.backend.connector.rest.dto.VersionDto
 import com.tngtech.apicenter.backend.connector.rest.mapper.VersionDtoMapper
 import com.tngtech.apicenter.backend.domain.handler.VersionHandler
 import org.springframework.web.bind.annotation.DeleteMapping
 import org.springframework.web.bind.annotation.GetMapping
 import org.springframework.web.bind.annotation.PathVariable
 import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestMethod
+import org.springframework.web.bind.annotation.RequestHeader
 import org.springframework.web.bind.annotation.RestController
+import org.springframework.http.MediaType
 import java.util.UUID
+import javax.servlet.http.HttpServletResponse
+import mu.KotlinLogging
+
+private val logger = KotlinLogging.logger {}
 
 @RestController
 class VersionController constructor(private val versionHandler: VersionHandler, private val versionDtoMapper: VersionDtoMapper) {
 
-    @GetMapping("/specifications/{specificationId}/versions/{version}")
-    fun findVersion(@PathVariable specificationId: UUID, @PathVariable version: String): VersionDto = versionDtoMapper.fromDomain(versionHandler.findOne(specificationId, version))
+    @RequestMapping("/specifications/{specificationId}/versions/{version}",
+        produces = ["application/json", "application/yml"],
```

Comment:
There should be constants for JSON, at least. Maybe you also can add one for yaml, somewhere?